src/main/java/hw_4/Task2.java

```diff
@@ -0,0 +1,31 @@
+package hw_4;
+
+import java.util.Scanner;
+
+public class Task2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter a string of integers separated by commas:");
+        String input = scanner.nextLine();
+
+        String[] numberStrings = input.split(",");
+        int[] numbers = new int[numberStrings.length];
+
+        for (int i = 0; i < numberStrings.length; i++) {
+            numbers[i] = Integer.parseInt(numberStrings[i]);
+        }
+
+        int min = Integer.MAX_VALUE;
+        int max = Integer.MIN_VALUE;
+
+        for (int number : numbers) {
+            min = Math.min(min, number);
+            max = Math.max(max, number);
+        }
+
+        System.out.println("Мінімальне число: " + min);
+        System.out.println("Максимальне число: " + max);
+
```

Comment:
The code doesn't work
![image](https://github.com/user-attachments/assets/85a6bd69-2a99-4aa1-bec3-7a0f6d06d249)

Reply:
Fixed. The code didn't work because I had a condition to separate the numbers with "commas", so it didn't work with "spaces". Now I have fixed it to put "spaces" between numbers when entering a series of numbers from the console.